### PR TITLE
Add a footer to each page

### PR DESCRIPTION
**Pull request by mridubhatnagar**
The solution is a fix for issue #76 

App List Page

applist

App Details Page

Screenshot from 2020-10-11 18-21-05

Table Page

Screenshot from 2020-10-11 18-21-49

### CODE REVIEW

**Comment by mridubhatnagar** [comment]
Hi @wlach, I have raised the PR for issue #76 but I am unable to figure out why the circle ci builds are failing.

**Comment by DawoodShahat** [comment]
Make sure to run ```npm run format``` to remove any formatting issue.

Also build it locally by ```npm run build``` to check circleci build before pushing.

William is having a weekend off, and will be back tomorrow.

Hope this helps 😀

**Comment by mridubhatnagar** [comment]
Hi @DawoodShahat, thank you for responding. 

Though I did run the npm run format command. It ran successfully and showed no errors.

File: src/pages/AppDetail.svelte [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <script>
+  import Footer from "../components/Footer.svelte";
```
Comment (DawoodShahat) [inline src/pages/AppDetail.svelte:2]: eslint requires you to add a new line after the import.

Screenshot 2020-10-11 at 7 49 38 PM

Add a new line after the import and you are good to go.

Comment (DawoodShahat) [inline src/pages/AppDetail.svelte:2]:
```suggestion
  import Footer from "../components/Footer.svelte";
  
```

Comment (mridubhatnagar) [reply src/pages/AppDetail.svelte:2]: Hey, thank you for showing the error. However, where did you find this error? I mean did you click the circle ci icon? I could reach here. Build still seems to fail after adding a new line after import too.

Comment (DawoodShahat) [reply src/pages/AppDetail.svelte:2]: Yes, I checked why the build failed by clicking the icon. 

I reproduced this linting error locally by removing the new line after the imports.

You can read about this issue more here
[https://github.com/sveltejs/eslint-plugin-svelte3/issues/41](https://github.com/sveltejs/eslint-plugin-svelte3/issues/41)

It looks like it is a formatting issue,
Screenshot 2020-10-11 at 9 45 59 PM


Try it with  ```npm run format```

**Comment by meghajain-1711** [comment]
> Hi @DawoodShahat, thank you for responding.
> 
> Though I did run the npm run format command. It ran successfully and showed no errors.

You can run npm test after you run npm format, It shows the same information as the CircleCI would do later. 
It will show the exact error log there.
Hope this helps :)

File: src/pages/TableDetail.svelte [hunk 2]
```diff
@@ -48,3 +49,5 @@
 
   <SchemaViewer app={params.app} nodes={data.schema} />
 {/await}
+
+<Footer />
```
Comment (robhudson) [inline src/pages/TableDetail.svelte:53]: Instead of adding the `Footer` to every page, you could instead add it once to the `src/App.svelte` file.

**Review by wlach — CHANGES_REQUESTED** [review]
In general this looks very good! There is a bit of an issue with regards to CSS properties, where some things were transferred over from GLAM (which doesn't use tailwind) that is implemented in a different way here. Feel free to ask on the chat if you have more questions.

While we're here, I'd also like to see a story for this element (see the section I just added to the README: https://github.com/mozilla/glean-dictionary#storybook). You can model it off of the existing story for the schema viewer:

https://github.com/mozilla/glean-dictionary/blob/main/stories/schemaviewer.stories.js

Note that the stories will look a little weird until #74 is fixed (which will hopefully be soon). As long as the content looks right I'm happy.

**Comment by wlach** [comment]
Hi @mridubhatnagar, did you want to continue working on this? If not, we might want to finish it up for you so we can get this feature landed.

**Comment by wlach** [comment]
Wound up moving this to: https://github.com/mozilla/glean-dictionary/pull/244

Thanks again @mridubhatnagar for all your hard work here.

**Pull request closed without merging**